### PR TITLE
{SQL} fix az sql server ad-admin update , az sql server ad-admin create calls

### PR DESCRIPTION
**Pull request by amolagar5**
**History Notes:**  
(Fill in the following template if multiple notes are needed, otherwise PR title will be used for history note.)

[Component Name 1] (BREAKING CHANGE:) (az command:) make some customer-facing change.  
[Component Name 2] (BREAKING CHANGE:) (az command:) make some customer-facing change.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
add to S167

File: src/azure-cli/azure/cli/command_modules/sql/_params.py [hunk 1]
```diff
@@ -1105,6 +1105,14 @@ def _configure_security_policy_storage_params(arg_ctx):
                 'sid',
             ])
 
+    with self.argument_context('sql server ad-admin update') as c:
+        # Update args that will be used to build up the ServerAzureADAdministrator object
+        create_args_for_complex_type(
+            c, 'properties', ServerAzureADAdministrator, [
+                'login',
+                'sid',
```
Comment (jaredmoo) [inline src/azure-cli/azure/cli/command_modules/sql/_params.py]: what happened to tenant_id?

File: src/azure-cli/azure/cli/command_modules/sql/tests/latest/test_sql_commands.py [hunk 2]
```diff
@@ -727,37 +727,44 @@ def test_sql_server_connection_policy(self, resource_group, resource_group_locat
                      checks=[JMESPathCheck('connectionType', type)])
 
 
-class AzureActiveDirectoryAdministratorScenarioTest(LiveScenarioTest):
+class AzureActiveDirectoryAdministratorScenarioTest(ScenarioTest):
     #  convert to ScenarioTest and re-record when ISSUE #6011 is fixed
```
Comment (jaredmoo) [inline src/azure-cli/azure/cli/command_modules/sql/tests/latest/test_sql_commands.py]: Is #6011 fixed?

File: src/azure-cli/azure/cli/command_modules/sql/custom.py [hunk 3]
```diff
@@ -2082,15 +2081,14 @@ def server_ad_admin_update(
     '''
     Updates a server' AD admin.
     '''
-
+
```
Comment (jaredmoo) [inline src/azure-cli/azure/cli/command_modules/sql/custom.py]: Nit: remove whitespace :)

File: src/azure-cli/azure/cli/command_modules/sql/tests/latest/test_sql_commands.py (hunk 2 above)
Comment (jaredmoo) [reply src/azure-cli/azure/cli/command_modules/sql/tests/latest/test_sql_commands.py]: If yes then please remove comment

**Comment by haroldrandom** [comment]
Facing an ADO error for now, will refresh any time if ADO available

**Comment by haroldrandom** [comment]
@amolagar5 CI is back, please take care of the code style problem.

**Comment by haroldrandom** [comment]
@amolagar5 CI is back, please take care of the code style problem.

**Comment by amolagar5** [comment]
@haroldrandom Fixed the styling issues, please take a look when possible.

**Pull request merged**